saas/views/metrics.py

```diff
@@ -166,9 +166,13 @@ def get_context_data(self, **kwargs):
                           "unit": "$", "table": account_table}
         data['customers'] = {"title": "Customers",
                              "table": customer_table, "extra": customer_extra}
-        context.update({"title": "Revenue Metrics",
-            "data": data,
-            "data_json": json.dumps(data, cls=DjangoJSONEncoder)})
+        context.update({
+            "title": "Revenue Metrics",
+            "tables": [
+                {"title": "Amount", "unit": "$", "key": "amount", "active": True},
```

Comment:
From where do I get this information? I see that the `Charge` object has a `unit` attribute, and that the `Organization.withdraw_funds` has a hard-coded variable `funds_unit` that is set to the string `usd`. Otherwise, I don't see any database fields that would indicate the currency type.